analyses/transcriptomic-dimension-reduction/01-transcriptomic-analysis-prep.R

```diff
@@ -304,75 +316,241 @@ transposed_kallisto_data <- t(exp_kallisto)
 #### RSEM ----------------------------------------------------------------------
 
 # Run `dimension_reduction_wrapper` function using PCA
+rsem_pca_aligned_scores_poly <-
```

Comment:
Probably call this `rsem_pca_aligned_scores_polyA` as there are many kinds of poly, and I was confused for a sec. (see also other places this appears)